### PR TITLE
Prepare prow for CDI

### PR DESCRIPTION
**Pull request by rmohr**

### CODE REVIEW

File: github/ci/prow/files/jobs/containerized-data-importer/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+filters:
+  ".*":
+    reviewers:
+      - awels
+      - mhenriks
```
Comment (awels) [inline github/ci/prow/files/jobs/containerized-data-importer/OWNERS]: can we add aglitke as well?

Comment (rmohr) [reply github/ci/prow/files/jobs/containerized-data-importer/OWNERS]: Done

**Comment by rmohr** [comment]
@awels I made all jobs optional, so that we can merge and test independently.

**Pull request merged**